File: 0645-set-mismatch/0645-set-mismatch.cpp
class Solution {
public:
    vector<int> findErrorNums(vector<int>& nums) {
    
    vector<int> ans(2);
  
    vector<int> count(nums.size() + 1, 0);
    

    for (int i = 0; i < nums.size(); i++) {
        count[nums[i]]++;
    }
    
    
    for (int i = 1; i <= nums.size(); i++) {
        if (count[i] == 2) {
            ans[0] = i;
        } else if (count[i] == 0) {
            ans[1] = i; 
        }
    }
    
    return ans;
    }
};

File: 0645-set-mismatch/0645-set-mismatch.py
class Solution(object):
    def findErrorNums(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        ans = [None] * 2
        count = [0] * (len(nums) + 1)
        n = len(nums)

        for i in range(0,n):
            count[nums[i]]+=1
        
        
        for i in range(0, n+1):
            if (count[i] == 2):
                ans[0] = i
            
            elif (count[i] == 0):
                ans[1] = i; 
            
        
        
        return ans